module.exports = {
  INTERNAL_ERROR:
    "Ocorreu um erro n√£o esperado. Visite os logs para mais detalhes.",
};
